theme/less/specific/reuse.less

```diff
@@ -1,14 +1,12 @@
 // Generic reuse cards
 .reuse-card {
-    width: 285px;
+    width: 100%;
+    max-width: 350px;
     .reuse-image {
         border-radius: @spacing-md;
         height: 480px;
         object-fit: cover;
         object-position: center center;
-    }
-
-    &.reuse-card--fluid {
-        width: 100%;
+        // aspect-ratio: 3 / 5; // We can use an aspect-ratio instead of a fixed height when
```

Comment:
when ... ? (suspense :p)

Reply:
We can use an aspect-ratio instead of a fixed height when we update our list of supported browsers.
This will not work on IE 11 and is only working on 82% of browsers on 01/2022.
Updated :)